front/src/modules/simulationResult/components/SimulationResultsMap/RenderItinerary.tsx

```diff
@@ -11,14 +13,36 @@ interface RenderItineraryProps {
 export default function RenderItinerary(props: RenderItineraryProps) {
   const { geojsonPath, layerOrder } = props;
 
-  const paint = {
-    'line-width': 3,
-    'line-color': '#82be00',
+  const paintBackgroundLine = {
+    'line-width': 4,
+    'line-color': '#EDF9FF',
   };
 
+  const paintLine = {
+    'line-width': 1,
```

Comment:
![Capture d’écran 2024-10-31 à 12 27 19](https://github.com/user-attachments/assets/4eaed522-a8ec-4a86-b888-b688d4e91575)

```suggestion
    'line-width': 3,
```

Reply:
I have something else in the mockup:  
<img width="273" alt="image" src="https://github.com/user-attachments/assets/d65abfef-4f6f-478e-86ad-e3096ad332f6">

Reply:
Oh right, I looked at the wrong map mockup